### PR TITLE
Add esy to client where lsp does not run by default

### PR DESCRIPTION
**Pull request by maxlibin**
Fix lsp does not start out of box, adding esy file should solve the problem.

### CODE REVIEW

**Comment by feihong** [comment]
@maxlibin Please also add directions to README to set up esy

**Comment by maxlibin** [comment]
Right, added a simple instruction on adding esy https://github.com/Sketch-sh/sketch-sh/pull/310/files#diff-8822179a8da757fde968cf80a9162e1c1d6f8cc170509cfbdea10be03b7e7d20R61

File: client/esy.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "name": "sketch.sh",
+    "dependencies": {
+        "@opam/reason": "*"
```
Comment (jchavarri) [inline client/esy.json:4]: Don't we need `@opam/ocaml-lsp-server` as dependency? How does it work otherwise?

**Review by jchavarri — COMMENTED** [review]
@maxlibin thanks for this! I am not sure I understand how it works, should not we add `ocaml-lsp-server` as dep?

**Comment by maxlibin** [comment]
I can't seems to get esy working on this package, I will close this PR, which LSP you guys using?

**Comment by jchavarri** [comment]
@maxlibin I am using an opam global switch set in 4.06 version of the OCaml compiler, but we should have instructions and commands to set up opam local switch better. 

I will create a PR with proposal.

**Pull request closed without merging**